nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,74 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+router.get("/", async (request, response) => {
+  try {
+    console.log("in/api/meals");
+    // variable for all the querys I am using so that I can check if one of the requested queryes from the user doesn't match/exists
+    const querys = ["maxPrice", "title", "createdAfter", "limit"];
```

Comment:
Nice idea, I like it! Maybe then you could also return a meaningful message if one of the query parameters doesn't exist? Eg. "parameter is not supported"